.github/nightly_matrix.php

```diff
@@ -76,17 +76,23 @@ function get_matrix_include(array $branches) {
 function get_windows_matrix_include(array $branches) {
     $jobs = [];
     foreach ($branches as $branch) {
+        $crt = $branch['name'] === 'master' ? 'vs17' : 'vs16';
```

Comment:
Well, could probably do the following right away:
```suggestion
        $crt = ($branch['name'] === 'master' || version_compare($branch['name'], 'PHP-8.4', '>=')) ? 'vs17' : 'vs16';
```